js/src/evaluation/evaluate_comparative.ts

```diff
@@ -69,16 +69,29 @@ async function loadTraces(
   return results;
 }
 
+/** @deprecated Use ComparativeEvaluatorNew instead: (args: { runs, example, inputs, outputs, referenceOutputs }) => ... */
+type ComparativeEvaluatorOld = (
```

Comment:
I think you may still need to export this for TypeScript reasons

Reply:
And how about `ComparativeEvaluatorLegacy` for this and `_ComparativeEvaluator` for this new type?